src/main/java/org/gridsuite/study/server/repository/rootnetwork/RootNetworkRepository.java

```diff
@@ -35,4 +35,8 @@ public interface RootNetworkRepository extends JpaRepository<RootNetworkEntity,
     Optional<RootNetworkEntity> findByNameAndStudyId(String name, UUID studyUuid);
 
     Optional<RootNetworkEntity> findByTagAndStudyId(String tag, UUID studyUuid);
+
+    Optional<RootNetworkEntity> findByIdAndName(UUID rootNetworkUuid, String rootNetworkName);
+
+    Optional<RootNetworkEntity> findByIdAndTag(UUID rootNetworkUuid, String rootNetworkName);
```

Comment:
This doesn't make sense.
Id is unique in RootNetworkEntity then why adding a where condition on the name or tag.
By the way, maybe you could use `findByNameAndStudyId` or `findByTagAndStudyId` variant ?
Or just use the findById if you got the `rootNetworkUuid`

Reply:
in another PR